src/VisualStudio/Core/Def/Commands.vsct

```diff
@@ -322,6 +322,16 @@
           <CommandName>AnalysisScopeEntireSolution</CommandName>
         </Strings>
       </Button>
+      <Button guid="guidRoslynGrpId" id="cmdidAnalysisScopeNone" priority="500" type="Button">
+        <Parent guid="guidRoslynGrpId" id="grpAnalysisScopeItems" />
+        <CommandFlag>DefaultDisabled</CommandFlag>
+        <Strings>
+          <ButtonText>&amp;None</ButtonText>
+          <CanonicalName>None</CanonicalName>
+          <LocCanonicalName>None</LocCanonicalName>
+          <CommandName>AnalysisScopeNone</CommandName>
+        </Strings>
```

Comment:
what si the impact of this?

Reply:
This is just adding to existing context menu commands that @sharwell recently added to Solution explorer so that users can override the global option value in Tools Options with a per-solution option value: 
![image](https://user-images.githubusercontent.com/10605811/150132991-72157102-bcaa-4b9a-b84c-357ca94a28dd.png)